Source/Scene/GltfImageLoader.js

```diff
@@ -293,7 +293,7 @@ function loadImageFromBufferTypedArray(typedArray) {
 const ktx2Regex = /(^data:image\/ktx2)|(\.ktx2$)/i;
 
 function loadImageFromUri(resource) {
-  const uri = resource.url;
+  const uri = resource._url;
```

Comment:
```suggestion
  const uri = resource..getUrlComponent(false, true);
```

I think using using the [`resource.getUrlComponent`](https://github.com/CesiumGS/cesium/blob/main/Source/Core/Resource.js#L566) function would also do the same job without having to access a private variable. That's basically what the public getter for the `url` property is using under the hood anyway: https://github.com/CesiumGS/cesium/blob/4aac68a5d6b3603e13c3885b526b94a78b79f429/Source/Core/Resource.js#L469-L472

Reply:
It works. Your method is better.